python/server/qgsserverrequest.sip

```diff
@@ -0,0 +1,76 @@
+/***************************************************************************
+                          qgsserverrequest.h
+
+  Define ruquest class for getting request contents
+  -------------------
+  begin                : 2016-12-05
+  copyright            : (C) 2016 by David Marteau
+  email                : david dot marteau at 3liz dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+/**
+ * \ingroup server
+ * QgsServerRequest
+ * Class defining request intreface passed to services QgsService::executeRequest() method
+ *
+ *  Note about design: this intreface must be passed along to python and thus signatures methods must be 
+ *  compatible with pyQGS/pyQT api and rules.
+ */
+class QgsServerRequest
```

Comment:
How about headers sent with the request... it might be useful to have them available...?